desktop/libs/aws/src/aws/s3/s3fs.py

```diff
@@ -220,6 +220,11 @@ def _stats(self, path):
 
   @staticmethod
   def _stats_key(key, fs='s3a'):
+    if key.size == 0:                                  # workaround for ozone s3 connector
```

Comment:
Same lets add more verbose comment, what is this workaround doing for Ozone S3 connector?